app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -509,38 +510,62 @@ public void onClick(View v) {
         switch (v.getId()) {
             case R.id.button_channel_parameters_os:
                 replaceFragment(R.id.layout_dock_os2, channelParametersFragment, "ChannelParametersFragment");
+                clearTextBackgroundColor();
+                channelParametersTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.tv_channel_parameters_os:
                 replaceFragment(R.id.layout_dock_os2, channelParametersFragment, "ChannelParametersFragment");
+                clearTextBackgroundColor();
+                channelParametersTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.button_timebase_os:
                 replaceFragment(R.id.layout_dock_os2, timebaseTriggerFragment, "TimebaseTiggerFragment");
+                clearTextBackgroundColor();
+                timebaseTiggerTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.tv_timebase_tigger_os:
                 replaceFragment(R.id.layout_dock_os2, timebaseTriggerFragment, "TimebaseTiggerFragment");
+                clearTextBackgroundColor();
+                timebaseTiggerTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.button_data_analysis_os:
                 replaceFragment(R.id.layout_dock_os2, dataAnalysisFragment, "DataAnalysisFragment");
+                clearTextBackgroundColor();
+                dataAnalysisTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.tv_data_analysis_os:
                 replaceFragment(R.id.layout_dock_os2, dataAnalysisFragment, "DataAnalysisFragment");
+                clearTextBackgroundColor();
+                dataAnalysisTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.button_xy_plot_os:
                 replaceFragment(R.id.layout_dock_os2, xyPlotFragment, "XYPlotFragment");
+                clearTextBackgroundColor();
+                xyPlotTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
 
             case R.id.tv_xy_plot_os:
                 replaceFragment(R.id.layout_dock_os2, xyPlotFragment, "XYPlotFragment");
+                clearTextBackgroundColor();
+                xyPlotTextView.setBackgroundColor(Color.parseColor("#c72c2c"));
                 break;
         }
     }
 
+    @SuppressLint("ResourceType")
+    private void clearTextBackgroundColor() {
+        channelParametersTextView.setBackgroundColor(Color.parseColor("#7A7A7A"));
```

Comment:
Hello @Vikum94 ! Wouldn't it be better to move the hardcoded colors to `colors.xml` and call from there?

Reply:
Ok. I'll change it. Thanks